libs/providers/flagd/src/e2e/setup-in-process-provider.ts

```diff
@@ -7,10 +7,13 @@ const FLAGD_NAME = 'flagd Provider';
 // register the flagd provider before the tests.
 console.log('Setting flagd provider...');
 OpenFeature.setProvider(
+  'e2e',
   new FlagdProvider({ cache: 'disabled', resolverType: 'in-process', host: 'localhost', port: 9090 }),
 );
-assert(
-  OpenFeature.providerMetadata.name === FLAGD_NAME,
-  new Error(`Expected ${FLAGD_NAME} provider to be configured, instead got: ${OpenFeature.providerMetadata.name}`),
-);
+OpenFeature.setProvider('unstable', new FlagdProvider({ resolverType: 'in-process', host: 'localhost', port: 9091 }));
+// TODO: update with correct assertions once we have ability to get providerMetadata for any provider
+// assert(
+//   OpenFeature.providerMetadata.name === FLAGD_NAME,
+//   new Error(`Expected ${FLAGD_NAME} provider to be configured, instead got: ${OpenFeature.providerMetadata.name}`),
+// );
```

Comment:
So we could run these tests in parallel, I started using a dedicated named provider `'e2e'` for this, but when I did that, I realized we can only query the metadata of the default provider. We need to add a feature to get named provider metadata to the js-sdk then fix this.